addons/hw_posbox_homepage/controllers/main.py

```diff
@@ -104,8 +104,51 @@ def index(self):
         else:
             return homepage_template.render(self.get_homepage_data())
 
-    @http.route('/list_handlers', type='http', auth='none', website=True)
-    def list_handlers(self):
+    @http.route('/list_handlers', type='http', auth='none', methods=['GET', 'POST'], website=True, csrf=False)
```

Comment:
@xmo-odoo this is on the IoT box, there is login nor user system set. Except if you mean a hidden empty dummy csrf hidden input ?

Reply:
indeed I'm stupid, I'll see if it is possible to do this on the IoT (I do guess csfr token are stored outside of the ram which is problematic as there is no database and the system is mounted on read-only)

Reply:
Note:  `csrf_token()` looks to need the "database secret" which does not exists in IoT (as there is no database)

Reply:
@xmo-odoo 
In theory it could work both way, for simplicity sake we use it in HTTP most of the time but we expect Odoo's clients to use it in HTTPS.

Do you think it would be relevant to do it in this PR ? Most of the routes in the IoT box are in `csrf=false` system, it might be better - in my opinion - to think over a whole refactoring of the IoT routes security (with a custom csrf token using the password hash of the IoT box for instance instead of the database secret ). What do you think ?

Reply:
@xmo-odoo I discussed on it with the IoT guru, according to him there is already discussion and a draft PR (in security) on the matter but it's a bit on hold for now.